cloudmarker/events/azrdbmssslenforcementevent.py

```diff
@@ -0,0 +1,104 @@
+"""Microsoft Azure RDBMS SSL Enforcement Event.
+
+This module defines the :class:`AzRDBMSSLEnforcementEvent` class that
+identifies RDBMS servers which have SSL connection enforcement
+disabled. This plugin works on the  properties found in the ``ext``
+bucket of ``rdbms`` records.
+"""
+
+
+import logging
+
+from cloudmarker import util
+
+_log = logging.getLogger(__name__)
+
+
+class AzRDBMSSSLEnforcementEvent:
+    """Az RDBMS SSL enforcement event plugin."""
+
+    def __init__(self):
+        """Create an instance of :class:`AzRDBMSSLEnforcementEvent`."""
+
+    def eval(self, record):
+        """Evaluate Azure RDBMS servers for SSL connection enforcement.
+
+        Arguments:
+            record (dict): A RDBMS record.
+
+        Yields:
+            dict: An event record representing an RDBMS server where SSL
+            connection enforcement is disabled
+
+        """
+        com = record.get('com', {})
+        if com is None:
+            return
+
+        if com.get('record_type') != 'rdbms':
+            return
+
+        if com.get('cloud_type') != 'azure':
+            return
+
+        ext = record.get('ext', {})
+        if ext is None:
+            return
+
+        if ext.get('record_type') not in ['mysql_server',
+                                          'postgresql_server']:
+            return
+
+        if ext.get('ssl_connection_enabled', False) is False:
```

Comment:
The idiomatic way to write this is:

```
if not ext.get('ssl_connection_enabled'):
```

Reply:
I want to make generate an event only if the value of `ssl_connection_enabled ` is False.
`if not ext.get('ssl_connection_enabled'):
 `
may evaluate to True if the value of `ssl_connection_enabled`  is anything other than `False` e.g. if `ssl_connection_enabled` is `None` the above condition will be True

Reply:
You are right. I wrote it wrong. I meant to say,

```
if not ext.get('ssl_connection_enabled', False):
```

Does this make sense?

Reply:
`if not ext.get('ssl_connection_enabled', False):` evaluates to `True` if the value of `ssl_connection_enabled` is anything other than `False` (e.g. `None`) or the key `ssl_connection_enabled` is missing from the `ext` bucket

Reply:
You are right again. I am getting confused with the case of `None` as a value. My